build/target-repository/docs/how_to_create_custom_fileprocessor.md

```diff
@@ -0,0 +1,208 @@
+# Create your own custom FileProcessor
+
+This section is all about creating your custom specific FileProcessor.
+If you don´t know the concept of FileProcessors in the context of Rector, have a look at [Beyond PHP - Entering the realm of FileProcessors](beyond_php_file_processors.md)
+
+Most of the examples starting with a rather contrived example, let´s do it the same.
+
+Imagine you would like to replace the sentence "Make america great again" to "Make the whole world a better place to be" in every file named dumb_trump.txt.
```

Comment:
Not sure if I would use Trump or any politician in examples :) (not Trump's fan)

Reply:
@sabbelasichon have you pushed changes? I still see DumpTrump and only one commit for this PR